SU-SOE/soe_basic
# D8CORE-3269: fixing the button padding on SOE

# READY FOR REVIEW

# Summary
- Removed button padding that was unequal per design and pm. It is an existing bug but now noticeable with the external links module

# Review By (Date)
- 1/10

# Urgency
- low

# Steps to Test

1. Pull in this change.
2. Create or look at button on SOE.
3. Verify the padding bottom is 11 and top is 10.5. This matches what we have on Stanford Basic.

# Affected Projects or Products
- SOE

# Associated Issues and/or People
- D8CORE-3269

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
